bridge/opencensus/trace.go

```diff
@@ -26,10 +26,21 @@ import (
 // NewTracer returns an implementation of the OpenCensus Tracer interface which
 // uses OpenTelemetry APIs.  Using this implementation of Tracer "upgrades"
 // libraries that use OpenCensus to OpenTelemetry to facilitate a migration.
+//
+// Deprecated: Use InstallTraceBridge instead.
 func NewTracer(tracer trace.Tracer) octrace.Tracer {
 	return internal.NewTracer(tracer)
 }
 
+// InstallTraceBridge installs the OpenCensus trace bridge, which overwrites
+// the global OpenCensus tracer implementation. Once the bridge is installed,
+// spans recorded using OpenCensus are redirected to the OpenTelemetry SDK.
+func InstallTraceBridge(opts ...TraceOption) {
+	cfg := newTraceConfig(opts)
+	tracer := cfg.tp.Tracer(scopeName)
+	octrace.DefaultTracer = internal.NewTracer(tracer)
```

Comment:
Is it never the case that OC users want to use the tracer directly?

Reply:
The opencensus Tracer is an interface I added later to allow this bridge solution to work: https://github.com/census-instrumentation/opencensus-go/pull/1234.  I don't expect anyone actually ever uses it directly

Reply:
Gotcha :+1: